data_specification/utility_calls.py

```diff
@@ -72,6 +72,40 @@ def get_data_spec_and_file_writer_filename(
         application_run_time_report_folder, _DAT_TMPL.format(
             hostname, processor_chip_x, processor_chip_y, processor_id)))
 
+    # check if text reports are needed and if so initialise the report
+    # writer to send down to DSG
+    report_writer = get_report_writer(
+        processor_chip_x, processor_chip_y, processor_id,
+        hostname, report_directory="TEMP", write_text_specs=False)
```

Comment:
This seems to force the report_directory to TEMP.  Should it instead pass the report_directory?  Similarly with write_text_specs.

Reply:
oops!